### PR TITLE
build out rspec files and gems

### LINKED ISSUE #5: Add gems and rspec files

### PR DESCRIPTION
**Pull request by Steddy1Love**
## What I did:
Added gems, spec and rails helper files. Also added code to the files to make sure they will work when needed. I also created the circleci folder and config file for CI/CD

## Did this break anything?
- [x] No
- [ ] Yes
    [ ] This broke some stuff
    [ ] This broke everything

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Refactor(DRY-ing up/ reorganizing code, etc.)
- [ ] Super small fix (Corrected a typo, removed a comment, etc.)
- [ ] Skip all the other stuff and briefly explain the fix.

## Requested feedback: 
- [x] Yes
    I'd like feedback on if there are suggestions on gems more or less.
- [ ] No

## Checklist:
- [x] If this code needs to be tested, all tests are passing
- [x] The code runs locally
- [x] There are comments where clarification/ organization is needed
- [x] The code is DRY. If it's not, I tried my best
- [x] I reviewed my code before pushing

## Testing Changes
- [x] No Tests have been changed
- [ ] Some Tests have been changed
- [ ] All of the Tests have been changed(Please describe what in the world happened)

## What's next?
CI/CD

Close #5

### CODE REVIEW

File: .circleci/config.yml
Comment (NicoShanstrom) [inline .circleci/config.yml:1]: ok yeah lines up with lesson for ruby on rails circleCI

File: .github/workflows/fly-deploy.yml
Comment (NicoShanstrom) [inline .github/workflows/fly-deploy.yml:1]: not really sure how do make this file if i was to do it, was it just the wizard on https://fly.io/docs/rails/getting-started/existing/ and then just kept following the prompts or did you see how to manually create this file?

File: Gemfile
Comment (NicoShanstrom) [inline Gemfile:1]: yep add docker, we should learn about that at some point through this project.

File: fly.toml
Comment (NicoShanstrom) [inline fly.toml:1]: ok this is autogenerated when creating the fly app.

File: spec/rails_helper.rb
Comment (NicoShanstrom) [inline spec/rails_helper.rb:1]: yeah, looks like the usual setup for this to start.

**Review by NicoShanstrom — COMMENTED** [review]
looks good, just have some clarifying questions about the fly-deploy.yml and the .yml files. I am assuming those get populated when setting up the fly app with the wizard maybe this: fly launch and follow the wizard.

**Comment by NicoShanstrom** [comment]
the circle CI build-and-test all checks fail comes up, is this something we need to worry about at the moment and can merge and fix the issues through building the application more?

File: config/routes.rb
Comment (NicoShanstrom) [inline config/routes.rb:1]: get "/", to: "home#index" could be the problem that the circleCI is not working. ** Execute environment
bin/rails aborted!
SyntaxError: --> /home/circleci/project/config/routes.rb
syntax error, unexpected end-of-input
   1  Rails.application.routes.draw do
>  6    get "up" => "rails/health#show", as: :rails_health_check
> 11    get "/", 
  12  end
/home/circleci/project/config/routes.rb:12: syntax error, unexpected `end' (SyntaxError)

**Review by NicoShanstrom — CHANGES_REQUESTED** [review]
i think the routes file could be causing the issue.

**Review by Steddy1Love — COMMENTED** [review]
Submitting commit to fix the faux route to check if the server is deployed.

File: .circleci/config.yml
Comment (NicoShanstrom) [inline .circleci/config.yml:1]: ok the problem was that you needed to have the specific image from circleci.

**Review by NicoShanstrom — APPROVED** [review]

**Comment by NicoShanstrom** [comment]
Successful changes for passing circleCI and creating the schema, good work.

**Pull request merged**